lib/qesapdeployment.pm

```diff
@@ -2029,4 +2031,83 @@ sub qesap_az_create_sas_token {
     record_info('GENERATE-SAS', $cmd);
     return script_output($cmd);
 }
+
+=head2 qesap_terraform_clean_up_retry
+
+    qesap_terraform_clean_up_retry()
+
+    Do terraform cleanup only when ansible failed on 'Timed out waiting for last boot time check'
+
+=cut
+
+sub qesap_terraform_clean_up_retry {
+    my $command = 'terraform';
+
+    # Do not do 'ansibe' cleanup as if 'Timed out waiting for last boot time check' happened the SSH will be disconnected
+    # E.g., ansible SSH reports '"msg": "Timeout (12s) waiting for privilege escalation prompt: "'
+    # Terraform destroy can be executed in any case
+    record_info('Cleanup', "Executing $command cleanup");
+    my @clean_up_cmd_rc = qesap_execute(verbose => '--verbose', cmd => $command, cmd_options => '-d', timeout => 1200);
```

Comment:
verbose argument is not mind to be used like this
```
verbose => '--verbose'
```

but like this
```
verbose => 1
```

Reply:
Changed to `verbose => 1`.